src/test/java/com/github/digitopolis/socket/SocketCreatorTest.java

```diff
@@ -0,0 +1,25 @@
+package com.github.digitopolis.socket;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.net.Socket;
+
+import com.github.digitopolis.echoserver.socket.SocketCreator;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Instead of using the 'import all' star notation for these packages, it would be good to be explicit about what you want to pull in.
My IntelliJ always used to do this and I had to enable a setting to turn off import-all by default: https://intellij-support.jetbrains.com/hc/en-us/community/posts/206203659-Turn-off-Wildcard-imports-